documentation.html

```diff
@@ -63,19 +63,19 @@ <h3>Avatar</h3>
                 <h5>Examples of Round Image Avatar:</h5>
                 <div class="container">
                     <div class="avatar">
```

Comment:
this div looks like a wrapper for avatar, use class name such as avatar-wrapper or avatar-container